src/app/components/HabitPlant/HabitPlant.tsx

```diff
@@ -5,23 +5,23 @@ import plant_bloom from '../../assets/plant_bloom.png';
 import empty_pot from '../../assets/empty_pot.png';
 
 type HabitPlantProps = {
-  rating: number;
+  rating: string;
```

Comment:
There are only specific ratings possible.
Take a look at unions:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types

```suggestion
  rating: 'emptyPot' | 'freshPlant' | 'comingAlong';
```

You need this in `getImgSrc` and `getRating` too -> you can create a type for this:

```ts
type Rating = 'emptyPot' | 'freshPlant' | 'comingAlong';
```

Reply:
@lmachens I've moved the rating into the config file and added a rating type (see last commit). What do you think?

Reply:
It's awesome φ(゜▽゜*)♪